elasticsearch/files/elasticsearch.yml

```diff
@@ -205,22 +218,21 @@ index.number_of_replicas: {{ server.get('index', {}).get('replicas', 1) }}
 # communication. (the range means that if the port is busy, it will automatically
 # try the next port).
 
-# Set the bind address specifically (IPv4 or IPv6):
-#
-#
-network.bind_host: {{ server.bind.address }}
-http.port: {{ server.bind.port }}
-http.enabled: true
-# Set the address other nodes will use to communicate with this node. If not
-# set, it is automatically derived. It must point to an actual IP address.
+# Set both 'bind_host' and 'publish_host':
 #
+# network.host: 192.168.0.1
+
+{%- if server.get('bind', {}).address is defined %}
+network.host: {{ server.bind.address }}
```

Comment:
To be honest I don't see what can be the issue but yes maybe it is better to force string.